project/MiMa.scala

```diff
@@ -36,7 +36,9 @@ object MiMa extends AutoPlugin {
             else 0
 
           if (!projectName.contains("typed")) {
-            expandVersions(2, 5, firstPatchOf25 to latestPatchOf25)
+            // It appears something went wrong while releasing 2.5.18 which caused
+            // many unexpected static methods
```

Comment:
what does this mean? was that when scalac changed for the problems we had with Akka Extensions?
Such as these issues https://github.com/akka/akka/issues/25449 https://github.com/akka/akka/issues/25910

Maybe add reference to these issues in the comment?